processing/data_collection/gazette/spiders/mg_contagem.py

```diff
@@ -0,0 +1,51 @@
+import datetime as dt
```

Comment:
It is not a good practice to create a PR that contains different features developed. It will make code review difficult as the reviewers will not have full context about what they need to look in the code.

This PR is related to improvements on Docker. It should not include changes or spiders for new cities. Each new spider needs to be a separated and new PR.

Reply:
I think I had a problem with rebase. Because all of these commits are already on the main branch.

Reply:
I didn't noticed that. But I saw that happened before, so I needed to comment :-)

Reply:
Ooh indeed. It's a very important placement always.